### PR TITLE
Golangci lint

### PR DESCRIPTION
**Pull request by msehnout**
Rebased #1062

### CODE REVIEW

File: internal/jsondb/db_test.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ func TestDegenerate(t *testing.T) {
 		db := jsondb.New(dir, 0755)
 
 		// write-only file
-		err = ioutil.WriteFile(path.Join(dir, "one.json"), []byte("{"), 0644)
+		err = ioutil.WriteFile(path.Join(dir, "one.json"), []byte("{"), 600)
```
Comment (teg) [inline internal/jsondb/db_test.go:52]: Dropping the leading `0` changes this from octal to decimal.

**Review by teg — CHANGES_REQUESTED** [review]
Would love to merge this :) Found a minor issue.

File: internal/jsondb/db_test.go (hunk 1 above)
Comment (msehnout) [reply internal/jsondb/db_test.go:52]: I was wondering the same. Was it intentional @ondrejbudai ?

Comment (ondrejbudai) [reply internal/jsondb/db_test.go:52]: I think that gosec didn't like `0644` and preferred `0600`. `600` is definitely a mistake.

**Pull request closed without merging**